components/hdw-tft/hdw-tft.c

```diff
@@ -387,7 +399,7 @@ void initTFT(display_t * disp, spi_host_device_t spiHost, gpio_num_t sclk,
 
 #if defined(CONFIG_ST7735_160x80)
     ESP_ERROR_CHECK(esp_lcd_new_panel_st7735(io_handle, &panel_config, &panel_handle));
-#elif defined(CONFIG_ST7789_240x135) || defined(CONFIG_ST7789_240x240) || defined(CONFIG_ST7735_128x160)
+#elif defined(CONFIG_ST7789_240x135) || defined(CONFIG_ST7789_240x240) || defined(CONFIG_ST7735_128x160) || defined(CONFIG_GC9307_240x280)
```

Comment:
Haha, maybe at this point `esp_lcd_new_panel_st7789()` is just a straight up misnomer. Boo on Espressif for naming such a versatile function so specifically